app/react/Layout/DocumentCounter.tsx

```diff
@@ -19,13 +26,18 @@ export const DocumentCounter = (props: EntityCounterProps) => {
           </>
         )}
         <b> {entityListCount} </b> <Translate>shown of</Translate>
-        <b> {entityTotal} </b> <Translate>documents</Translate>
+        <b>
+          {' '}
+          {entityTotal}
+          {maxedTotalEntities === 'gte' && '+'}{' '}
+        </b>{' '}
```

Comment:
I'm not sure I like the structure of these `{' '}`: there is one inside the <b>, then one in the same line before the </b> and then one again after the </b>.  Can we control this with paddings on the CSS instead of this random spaces here?

Reply:
I think in this case is easier to add a space after _show of_ and another before _documents_ instead of CSS padding, since there are supposed to be a nbsp between this elements of the component.